spring-messaging/src/main/java/org/springframework/messaging/simp/stomp/StompDecoder.java

```diff
@@ -78,7 +78,7 @@ public MessageHeaderInitializer getHeaderInitializer() {
 	 * Decodes one or more STOMP frames from the given {@code ByteBuffer} into a
 	 * list of {@link Message Messages}. If the input buffer contains partial STOMP frame
 	 * content, or additional content with a partial STOMP frame, the buffer is
-	 * reset and {@code null} is returned.
+	 * reset and an empty list is returned.
```

Comment:
https://github.com/spring-projects/spring-framework/blob/01c62f86b35b39666c250fbc4a319579af450db5/spring-messaging/src/main/java/org/springframework/messaging/simp/stomp/StompDecoder.java#L112-L116

```
// decode() comment
	 * <p>If the given ByteBuffer contains only partial STOMP frame content and no
	 * complete STOMP frames, an empty list is returned
```
(Incorrect comment) Not related to this PR but, based on code I think we return `empty list ` instead of `null` on `decode()` so fix this comment~!